### PR TITLE
Support multiple versions of docs in documentation app

### PR DESCRIPTION
**Pull request by lfdebrux**
This PR lays the groundwork for hosting the v12 and v13 docs from this repo.

It isolates each folder in `/docs` into a separate app with separate assets and paths.

This is part of the work needed to solve #59.

### CODE REVIEW

File: docs/v12/documentation_routes.js [hunk 1]
```diff
@@ -19,7 +19,12 @@ router.all('*', function (req, res, next) {
 
 // Docs index
 router.get('/', function (req, res) {
-  res.render('index')
+  if (!req.originalUrl.endsWith('/')) {
```
Comment (nataliecarey) [inline docs/v12/documentation_routes.js]: This has a bug in it.  `http://localhost:3000/docs?abc=def` redirects to `http://localhost:3000/docs?abc=def/`.  I've tried this solution and it works for me:

```
router.get('/', function (req, res) {
  const [url, originalQueryString] = req.originalUrl.split('?')
  if (!url.endsWith('/')) {
    const queryString = originalQueryString ? `?${originalQueryString}` : ''
    // redirect to /docs/ so we can use relative hrefs safely
    res.redirect(`${url}/${queryString}`)
  } else {
    res.render('index')
  }
})
```

Comment (lfdebrux) [reply docs/v12/documentation_routes.js]: Good catch

Comment (lfdebrux) [reply docs/v12/documentation_routes.js]: Fixed

**Review by BenSurgisonGDS — APPROVED** [review]
Sorry, it's been a bit of long review, but I think it looks ok

**Pull request merged**